client/asset/btc/btc.go

```diff
@@ -47,10 +47,14 @@ const (
 	// The default fee is passed to the user as part of the asset.WalletInfo
 	// structure.
 	defaultFee = 100
+	// defaultFeeRateLimit is the default value for the feeratelimit.
+	defaultFeeRateLimit = 600
 	// defaultRedeemConfTarget is the default redeem transaction confirmation
 	// target in blocks used by estimatesmartfee to get the optimal fee for a
 	// redeem transaction.
 	defaultRedeemConfTarget = 2
+	// Smallest unit is 1 satoshi = 1e-8 BTC
+	smallestUnit = 1e8
```

Comment:
This is a little confusing. I see you're trying to use this in two different places, so I assume the reason this is `1e8` instead of `1e-8` like it says in the doc is to make that easier? Regardless, `1e8` is not `small`. It would be more clear to just hard code the `1e8` and `1e-5` in the two places you're using this new constant, I think.

Reply:
done, actually no need to use 1e-5 hard coded anywhere with last commit changes.